manifests/install.yaml

```diff
@@ -197,16 +197,19 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: argo
+  namespace: argo
```

Comment:
this looks wrong to me

Reply:
caused by https://github.com/argoproj/argo-workflows/blob/7dc6515ce1ef76475ac7bd2a7a3c3cdbe795a13c/manifests/cluster-install/kustomization.yaml

Reply:
introduced here: https://github.com/argoproj/argo-workflows/pull/5907